### ISSUE TITLE
terminate called after throwing an instance of 'Tango::DevFailed'

### ISSUE THREAD

**Issue by vallsv** [taurus-org/taurus#1086]
Hi,

I have created a very small Taurus application terminating this way.

```
terminate called after throwing an instance of 'Tango::DevFailed'
```
THis message is a guess print by Qt at the event loop termination. But i don't know.

I turn the python looging module to debug but there is nothing.

By chance do you have any idea how to debug this kind of things?

**Comment by vallsv** [comment]
I thought the logging config was propagated. I found i can enable the Taurus polling. But it do not help much. There is maybe other things i cna turn on?
```
MainThread     DEBUG    2020-03-02 11:58:18 TaurusPollingTimer[3000].Timer on _pollAttributes: Timer::start()
MainThread     DEBUG    2020-03-02 11:58:18,973 TaurusPollingTimer[3000].Timer on _pollAttributes: Timer::start()
TimerLoop 1    DEBUG    2020-03-02 11:58:18 TaurusPollingTimer[3000].Timer on _pollAttributes: Timer thread starting
TimerLoop 1    DEBUG    2020-03-02 11:58:18,974 TaurusPollingTimer[3000].Timer on _pollAttributes: Timer thread starting
MainThread     DEBUG    2020-03-02 11:58:18 TaurusPollingTimer[3000].Timer on _pollAttributes: Timer::stop()
MainThread     DEBUG    2020-03-02 11:58:18,974 TaurusPollingTimer[3000].Timer on _pollAttributes: Timer::stop()
terminate called after throwing an instance of 'Tango::DevFailed'
Aborted
```

**Comment by reszelaz** [comment]
Hi Valentin,
Do you have any of the devices down?
Are you using the latest taurus from develop (or at least with #1061 merged)?
I used to see this kind of errors (the GUI was not even starting) when there were devices not exported, but this got fixed with #1061.

**Comment by vallsv** [comment]
Thanks a lot.
It looks to be the case. A device is not there.
I still have no GUI displayed, but iv got a nice traceback.
I will try to see what i can do.
